### ISSUE TITLE
[All] Administrative - the boring bits

### ISSUE THREAD

**Issue by GonzoBright** [MattGuerrette/VixenEngine#8]
I have a few items that may need attention, unfortunately, to do with everything except the code, which happens to the the fun bit.

First, I was wondering whether it would be prudent to have a universal coding style written up, possibly added to a wiki of some sort, simply for consistency within the project. Additionally, the wiki may contain an upcoming features list.

Second, a mode of communication would be helpful in discussing minor items that aren't classified as issues for general communication, at the very least.

The items listed above are little things that will certainly prove useful in creating a project team, and creating an engine that will be almost indispensable for use in popular games, and in education. Cheers

**Comment by MattGuerrette** [comment]
Yes, i think at some point there should be an IRC channel, and mailing list created for the project to facilitate communication between the developers. As for coding standards, I will create a new CONTRIBUTING.md file and append a style guide to it.

**Comment by GonzoBright** [comment]
Fantastic. Let me know how I can help with this. Once the style guide is complete, I would be more than happy to run through each of the source files to check for compliance.

**Comment by MattGuerrette** [comment]
Sure thing. It might be a few days before I get it done. I am currently focused on re-evaluating the Lua scripting in Vixen.

**Comment by GonzoBright** [comment]
Fair enough, that sounds great. I would also like to ask whether you see the necessity for a 3D physics engine in the near future?

**Comment by MattGuerrette** [comment]
Yes, I was considering Bullet physics as the solution for Vixen, as it is open source. It is also used through the game industry as an alternative to Havok.

**Comment by GonzoBright** [comment]
That sounds fun. I'll look at that some at a later hour tonight.
